app/util/AWS.scala

```diff
@@ -24,6 +25,15 @@ class AWSConfig @Inject() (config: Configuration) {
 
   lazy val credProvider = new AWSCredentialsProviderChain(credProviders.flatten: _*)
 
+  lazy val sessionId: String = "session" + Math.random()
+
+  lazy val kinesisArn = config.getString("aws.kinesis.streamArn").getOrElse("")
+
+  lazy val atomsCredProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("composer"),
```

Comment:
@akash1810 I can't get it working actually it doesn't seem to be the way its done in other projects. Having thought about it, it is probably not the simplest way of doing it. Being able to read from the code/prod streams in this way would require adding the assume role arn to the configuration file and filling into the cloudformation. Copying composer credentials seems simpler. The composer credentials would not be needed to run the app, because there are kinesis streams for local development in the media-service account. Getting these credentials would only be needed for reading from the code/prod streams.